reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -945,6 +965,16 @@ function ReadableStreamDefaultReaderRead(reader, readRequest) {
   }
 }
 
+function ReadableStreamDefaultReaderRelease(reader) {
+  const e = new TypeError('Reader was released');
+  const readRequests = reader._readRequests;
+  reader._readRequests = [];
```

Comment:
This dance (lines 970-975) happens enough times it might be worth its own AO.

Reply:
Added `ReadableStreamDefaultReaderErrorReadRequests` and `ReadableStreamBYOBReaderErrorReadIntoRequests` helpers.